R/sulpher.R

```diff
@@ -90,7 +90,7 @@ calc_slv <- function(B_LU_BRP, B_SOILTYPE_AGR, B_AER_CBS,A_SOM_LOI,A_S_RT, D_BDS
   
     # calculate C-stock (kg/ ha) and CS ratio (sven: check unit)
     dt.arable[,D_OC := A_SOM_LOI * 100 * 100 * 0.3 * D_BDS * 0.01]
-    dt.arable[,A_CS_RAT := A_SOM_LOI * 10 / (A_S_RT * 0.001)]
+    dt.arable[,A_CS_RAT := A_SOM_LOI * 10 / (A_S_RT)]
   
```

Comment:
no this is incorrect. CS_RAT is the SOM-to-S ratio. Note that this is rather SOM stock than SOC, but anyhow, A_SOM_LOI times 10 results in units g/kg, so when dividing by A_S_RT * 0.001 the resulting unit is dimensionless.